operations/security_and_authentication/Audit_Logs.md

```diff
@@ -0,0 +1,65 @@
+# Audit Logs
+
+How to enable and disable audit logging for host and Kubernetes APIs.
+
+Audit logs are used to monitor the system and search for suspicious behavior.
+Host and Kubernetes API audit logging can be enabled to produce extra audit logs for analysis.
+Enabling audit logging is optional and will generate some load and data on the non-compute nodes \(NCNs\).
+Host and Kubernetes API audit logging is not enabled by default.
+
+## Procedure
+
+1. Enable or disable host and Kubernetes API audit logging.
+
+   The method for updating the audit log settings will vary depending on the state of the system.
+   Select one of the following options based on the installation status of the system:
+
+   * Install
+
+     During the install, audit logging is enabled or disabled by changing the CSI settings.
+     Use the following flags with the `csi config init` command to enable or disable audit logging.
+     Refer to `csi config init -h` for more information on using flags.
+
+     * `ncn-mgmt-node-auditing-enabled`: Set to `true` to enable host logging or to `false` to disable host logging.
+
+       ```console
+       initCmd.Flags().Bool("ncn-mgmt-node-auditing-enabled", true, "Enable management node auditing")
+       ```
+
+     * `k8s-api-auditing-enabled`: Set to `true` to enable Kubernetes API logging or to `false` to disable Kubernetes API logging.
+
+       ```console
+       initCmd.Flags().Bool("k8s-api-auditing-enabled", true, "Enable the kubernetes auditing API")
+       ```
```

Comment:
These snippets are modifying CSI source code, which would require rebuilding CSI.  This should probably look more like:
```
csi config init --k8s-api-auditing-enabled=true [other config init options]
```
Same for the other flag.

Should also maybe mention that these can be set in s`ystem_config.yaml`.  E.g. on redbull we're setting them to `false`, which is the default, but if we wanted to enable them this is where we'd do it:
```
redbull-ncn-m001-pit:/var/www/ephemeral/prep # grep audit system_config.yaml
k8s-api-auditing-enabled: false
ncn-mgmt-node-auditing-enabled: false
redbull-ncn-m001-pit:/var/www/ephemeral/prep #
```